.phan/config.php

```diff
@@ -45,6 +45,7 @@
 	"exclude_analysis_directory_list" => [
 		"vendor",
 		"htdocs/api/",
+                "test/unittests"
```

Comment:
Why are you excluding the unit tests?

Reply:
It will check the PHPUnit library, cause namespace issue.

Reply:
test/unittests/SinglePointLoginTest.php:44 PhanAccessMethodInternal Cannot access internal method \PHPUnit\Framework\MockObject\Builder\InvocationMocker::will() of namespace \PHPUnit\Framework\MockObject\Builder defined at vendor/phpunit/phpunit/src/Framework/MockObject/Builder/InvocationMocker.php:63 from namespace \

Reply:
I think we should solve the issue instead of excluding all the units tests from phan analysis. Phan can help us detect errors in the unit test code.

In case this is a specific issue for PHP Unit, I think we should try enabling the PHP Unit plugins for phan. https://github.com/phan/phan/tree/master/.phan/plugins#phpunitassertionpluginphp

I can easily add it to this PR (https://github.com/aces/Loris/pull/4964) or make a new one.

Reply:
@driusan @johnsaigle any ideas? I can't figure it out this namespace issue.

Reply:
I'll make a PR to enable the plugins and we'll see if it helps